modules/installation-vsphere-config-yaml.adoc

```diff
@@ -125,89 +93,53 @@ base and include the cluster name.
 ====
-endif::openshift-origin[]
-ifndef::restricted[]
-ifndef::openshift-origin[]
-<16> The pull secret that you obtained from {cluster-manager-url}. This pull secret allows you to authenticate with the services that are provided by the included authorities, including Quay.io, which serves the container images for {product-title} components.
-<17> The public portion of the default SSH key for the `core` user in
-{op-system-first}.
-endif::openshift-origin[]
-ifdef::openshift-origin[]
-<16> You obtained the {cluster-manager-url-pull}. This pull secret allows you to authenticate with the services that are provided by the included authorities, including Quay.io, which serves the container images for {product-title} components.
-<17> The public portion of the default SSH key for the `core` user in
+<15> The pull secret that you obtained from {cluster-manager-url}. By specifying the pull secre, you can authenticate with the services that are provided by the included authorities, including Quay.io, which serves the container images for {product-title} components.
+<16> The public portion of the default SSH key for the `core` user in
 {op-system-first}.
-+
-[NOTE]
-====
-For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
-====
-endif::openshift-origin[]
-endif::restricted[]
+endif::vsphere,vpshere-custom[]
 ifdef::restricted[]
-ifndef::openshift-origin[]
-<16> For `<local_registry>`, specify the registry domain name, and optionally the
-port, that your mirror registry uses to serve content. For example
-`registry.example.com` or `registry.example.com:5000`. For `<credentials>`,
-specify the base64-encoded user name and password for your mirror registry.
-<17> The public portion of the default SSH key for the `core` user in
-{op-system-first}.
+<14> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
 +
-[NOTE]
+[IMPORTANT]
 ====
-For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
```

Comment:
Why was this SSH note replaced with the FIPS important admonition?

Reply:
Line numbers changed based on the removed duplicate lines, but the notes still target the same callouts. Please let me know if I misunderstood the step out from the original content.